scripts/overrides/tei_to_solr_caseid.rb

```diff
@@ -99,6 +98,14 @@ def documents(x)
     people_role.each do |role, people|
       build_person_fields(x, role, people)
     end
+
+    # now figure out which document had the most recent date and use that as the
+    # caseid date (and dateDisplay)
+    dates = dates.compact.reject(&:empty?).map{|d| CommonXml.date_standardize(d)}
+    if dates.length > 0
+      x.field(dates.last, "name" => "date")
```

Comment:
Does `dates.compact...` sort them based on their values? I don't think it does so I'm worried that using `dates.last` here isn't actually getting the most recent date

Reply:
Discussed with Jess and she'll address in a commit to master :+1: